generate.sh

```diff
@@ -2,7 +2,59 @@
 
 srcdir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )
 
+# ask1() { local yn; echo -n "$* [Y/n] "; read -r -n 1 yn; [[ "$yn" =~ [Yy] ]] && echo && return 0; [ -z "$yn" ] && return 0; echo; return 1; }
+# if ask1 "Question?"; then ...; fi
```

Comment:
Maybe remove this, since we don't intend to make the script interactive.

Reply:
OK.